benchmark/compare.js

```diff
@@ -95,13 +95,12 @@ function run() {
         line = line.trim();
         if (!line)
           return;
-
-        var s = line.split(':');
-        var num = +s.pop();
+        var result = line.match(/: (\d*\.?\d*)/);
+        var num = +result[1];
         if (!num && num !== 0)
           return;
 
-        line = s.join(':');
+        line = line.substring(0, result.index);
```

Comment:
Is this backwards compatible? It probably also need to be able to parse the old output.